extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -54,6 +55,18 @@ class AmpFitText extends AMP.BaseElement {
     this.textContent_ = '';
   }
 
+  /** @override */
+  upgradeCallback() {
+    if (
+      // eslint-disable-next-line no-undef
```

Comment:
I think eslint not needed anymore here. See https://github.com/ampproject/amphtml/pull/32538/files

Reply:
Running `gulp lint` yields:
```
extensions/amp-fit-text/0.1/amp-fit-text.js
  61:7  error  'BENTO_AUTO_UPGRADE' is not defined  no-undef
```

Reply:
/cc @rsimha

Reply:
This is a case where both of you are correct 😃 The `eslint-disable` directive won't be necessary if you add `BENTO_AUTO_UPGRADE` to this file:

https://github.com/ampproject/amphtml/blob/327d021b18ad902711370dae815087fc70a29889/build-system/global-configs/experiments-const.json#L1-L4

Reply:
Got it, thanks!